webpack/utils/dateTimeHelpers.js

```diff
@@ -0,0 +1,7 @@
+import moment from 'moment';
```

Comment:
this broke katello nightly RPM builds:

```
node --max_old_space_size=2048 /builddir/build/BUILD/katello-4.4.0.pre.master/usr/share/foreman/node_modules/webpack/bin/webpack.js --config /builddir/build/BUILD/katello-4.4.0.pre.master/usr/share/foreman/config/webpack.config.js --bail --env.pluginName=katello
ModuleNotFoundError: Module not found: Error: Can't resolve 'moment' in '/builddir/build/BUILDROOT/tfm-rubygem-katello-4.4.0-0.1.pre.master.20211118150958git64ceeac.el7.noarch/opt/theforeman/tfm/root/usr/share/gems/gems/katello-4.4.0.pre.master/webpack/utils'
```

please fix ASAP.

Reply:
Rodger dodger!

Reply:
https://github.com/Katello/katello/pull/9797